### PR TITLE
Login-Register-page UI

### PR DESCRIPTION
**Pull request by tranvuminhtriet**

### CODE REVIEW

**Comment by namnguyenowt29** [comment]
Where is the sign-up form ?

**Comment by tranvuminhtriet** [comment]
> Where is the sign-up form ?

I'm doing sign-up form. I'll create other PR for it when it's done

File: src/app/features/auth/components/login-form/login-form.component.html [hunk 1]
```diff
@@ -1 +1,38 @@
-<p>login-form works!</p>
+<form
+  nz-form
+  [formGroup]="loginForm"
+  [nzLayout]="'vertical'"
+  (submit)="onSubmit()"
+>
+  <nz-form-item>
+    <nz-form-label nzFor="email"> Email </nz-form-label>
+    <nz-form-control nzErrorTip="Email is not valid">
```
Comment (namnguyenowt29) [inline src/app/features/auth/components/login-form/login-form.component.html:9]: No hardcode.
If we have > 2 validator, you need the corresponding error message.
Consider the example in antd form document

File: src/app/features/auth/components/login-form/login-form.component.html [hunk 2]
```diff
@@ -1 +1,38 @@
...
+        formControlName="email"
+        type="email"
+        id="email"
+        placeholder="Enter email"
+      />
+    </nz-form-control>
+  </nz-form-item>
+  <nz-form-item>
+    <nz-form-label nzFor="password"> Password </nz-form-label>
+    <nz-form-control nzErrorTip="Password is required">
```
Comment (namnguyenowt29) [inline src/app/features/auth/components/login-form/login-form.component.html:21]: No hardcode

File: src/app/features/auth/components/login-form/login-form.component.ts [hunk 3]
```diff
@@ -1,15 +1,30 @@
 import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-login-form',
   templateUrl: './login-form.component.html',
-  styleUrls: ['./login-form.component.scss']
+  styleUrls: ['../styles/form.component.scss']
```
Comment (namnguyenowt29) [inline src/app/features/auth/components/login-form/login-form.component.ts:7]: Don't move the scss file into another folder.

File: src/app/features/auth/components/login-form/login-form.component.ts [hunk 4]
```diff
@@ -1,15 +1,30 @@
...
 })
 export class LoginFormComponent implements OnInit {
 
+  isLoading = false;
+  loginForm: FormGroup;
```
Comment (namnguyenowt29) [inline src/app/features/auth/components/login-form/login-form.component.ts:12]: You can use this key word `!` , `loginForm !: FormGroup`
But DON'T abuse it.

File: src/app/features/auth/components/register-form/register-form.component.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
+<form
+  nz-form
+  [formGroup]="registerForm"
+  [nzLayout]="'vertical'"
+  (submit)="onSubmit()"
+>
+  <nz-form-item>
+    <nz-form-label nzFor="name"> Name </nz-form-label>
+    <nz-form-control nzErrorTip="Name is required">
```
Comment (namnguyenowt29) [inline src/app/features/auth/components/register-form/register-form.component.html:9]: No hardcode

File: src/app/features/auth/components/register-form/register-form.component.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        formControlName="name"
+        type="text"
+        id="name"
+        placeholder="Enter name"
+      />
+    </nz-form-control>
+  </nz-form-item>
+  <nz-form-item>
+    <nz-form-label nzFor="email"> Email </nz-form-label>
+    <nz-form-control nzErrorTip="Email is not valid">
```
Comment (namnguyenowt29) [inline src/app/features/auth/components/register-form/register-form.component.html:21]: No hardcode

**Pull request merged**